scripts/exec.js

```diff
@@ -0,0 +1,33 @@
+var exec = require('child_process').exec;
+
+var command_line;
+var environ;
+
+switch(process.argv[2]) {
+  case 'build':
+    command_line = 'webpack --config $npm_package_webpack --progress --colors';
```

Comment:
Yeah, I wondered about this one... Just put `'./node_modules/kcd-common-tools/shared/webpack.config.js'` in directly instead of `$npm_package_webpack`